aws-xray-recorder-sdk-sqs/src/main/java/com/amazonaws/xray/sqs/SQSMessageHelper.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.xray.sqs;
+
+import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+
+public final class SQSMessageHelper {
```

Comment:
XRaySQSMessageHelper?

Reply:
I think it makes sense to change the package name from SQS to lambda. But let's keep the class name concise.